### PR TITLE
geographic-web-services review

### PR DESCRIPTION
**Pull request by eblondel**

### CODE REVIEW

**Review by Robinlovelace — APPROVED** [review]
This looks great to me @eblondel many thanks. Will wait to see if @Nowosad or @jannes-m have other comments but this is good to go from my perspective and a clear improvement just in time for the second edition.

**Review by Nowosad — APPROVED** [review]

**Comment by Nowosad** [comment]
Looks good -- thank you a log @eblondel. I just made some small text changes. 

@Robinlovelace -- feel free to merge it.

File: 08-read-write-plot.Rmd [hunk 1]
```diff
@@ -196,18 +196,23 @@ This package also allows performing the opposite process called reverse geocodin
...
+Various open-source projects have adopted these protocols, such as [GeoServer](https://geoserver.org/) and [MapServer](https://mapserver.org/) for data handling, or [GeoNetwork](https://geonetwork-opensource.org/) and [PyCSW](https://pycsw.org/) for metadata handling, leading to standardization of queries. 
+Integrated tools for Spatial Data Infrastructures (SDI), such as [GeoNode](https://geonode.org/), [GeOrchestra](https://www.georchestra.org/) or [Examind](https://www.examind.com/) have also adopted these standard web-services, either directly or by using previously mentioned open-source tools.
 
 Like other web APIs, OWS APIs use a 'base URL', an 'endpoint' and 'URL query arguments' following a `?` to request data (see the [`best-practices-api-packages`](https://httr.r-lib.org/articles/api-packages.html) vignette in the **httr** package).
 
-There are many requests that can be made to a OWS service. Below examples illustrate how some requests can be made directly with **httr** or in easier way with the **ows4R** package (OGC Web-Services for R).
```
Comment (Robinlovelace) [inline 08-read-write-plot.Rmd]: :+1: for typo fix.

**Review by Robinlovelace — COMMENTED** [review]
This looks like a definite improvement. Will merge and make post-merge tweaks, probably as a PR to enable feedback on my changes. Many thanks!

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**